certif/app/components/session-supervising/candidate-in-list.js

```diff
@@ -9,6 +9,14 @@ export default class CandidateInList extends Component {
   @tracked isMenuOpen = false;
   @tracked isConfirmationModalDisplayed = false;
 
+  get checkboxShouldBeDisplayed() {
```

Comment:
suggestion: displayCheckbox()
Apres reflexion, on renvoi un boolean ici, donc ca serait bien d'avoir `is` comme prefix mais je n'ai pas trop d'idee

Reply:
Je peux mettre `isCheckboxDisplayed` mais je n'aime pas trop cette convention parce que je ne la trouve pas toujours pertinente : ici la fonction ne renvoie pas `true` si la checkbox est effectivement affichée, mais si elle devrait l'être (c.a.d si les conditions sont réunies pour qu'on veuille l'afficher). Donc mettre `is` plutôt que `shouldBe` peut être trompeur sur ce que fait la fonction.

Reply:
Oui, je ne suis pas non plus fan du  `isCheckboxDisplayed`, peut etre `isCheckboxToDisplay` ?